packages/database/prisma/schema.prisma

```diff
@@ -246,6 +248,33 @@ model TrackedKeyword {
   @@unique([userId, keyword, providerId]) // Prevents duplicate tracking
 }
 
+model Competitor {
```

Comment:
How do i know which integration a competitor belongs to?

Reply:
Would this allow us to have better grained control if we ever want to restrict number of competitor intgrations a user can have?